h2o-py/h2o/estimators/infogram.py

```diff
@@ -745,6 +762,23 @@ def total_information_threshold(self):
         information is the x-axis of the Core Infogram. Default is -1 which gets set to 0.1.
```

Comment:
Out of scope but I don't understand `Default is -1 which gets set to 0.1.`

Reply:
Also the first line is not correct since the default is -0.1.

`A number between 0 and 1 representing a threshold for total information, defaulting to 0.1.`

Reply:
cc: @hannah-tillman

Reply:
@hannah-tillman I understand it now.

It comes from here: 

https://github.com/h2oai/h2o-3/blob/master/h2o-admissibleml/src/main/java/hex/Infogram/Infogram.java#L185-L187

@wendycwong any reason why we not set it directly [here](https://github.com/h2oai/h2o-3/blob/master/h2o-admissibleml/src/main/java/hex/Infogram/InfogramModel.java#L59)? 

Nevertheless, its out of scope of this PR. @shaunyogeshwaran.

Reply:
@valenad1 @hannah-tillman 

The value of -1 or -0.1 is used to denote that the user has not set any value.  If the user has not set any value, we will set it to a default value of 0.1.  There is a reason that the code needs to know if the user set that value.  I cannot remember what it is now.